pkg/cli/admin/upgrade/upgrade.go

```diff
@@ -45,7 +45,7 @@ func New(f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.Command
 	o := NewOptions(streams)
 	cmd := &cobra.Command{
 		Use:     "upgrade --to=VERSION",
-		Short:   "Upgrade a cluster",
+		Short:   "Upgrade a cluster or adjust the update service channel",
```

Comment:
Suggestion `Upgrade a cluster or adjust the upgrade channel`

Reply:
Works for me.  Updated in 84daf9fa2 -> 0e9e0bf06.